src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs

```diff
@@ -544,7 +551,7 @@ internal IEnumerable<BluetoothGattDescriptor> GetDescriptors(BluetoothGattCharac
                 return true;
             };
 
-            int err = Interop.Bluetooth.BtGattCharacteristicForeachDescriptors(characteristic.GetHandle(), cb, IntPtr.Zero);
+            int err = Interop.Bluetooth.BtGattCharacteristicForeachDescriptors(characteristic.GetHandle(), _descriptorForeachCallback, IntPtr.Zero);
```

Comment:
`foreach` is not async method, so need not change.
anyway it also no matter

Reply:
Thank you for comment. I want to leave it as it is.